src/Firestore.elm

```diff
@@ -269,6 +285,20 @@ delete (Firestore config path) =
         }
 
 
+{-| Deletes a document. Will fail if document does not exist.
+-}
+deleteExisting : Firestore -> Task.Task Error ()
+deleteExisting (Firestore config path) =
+    Http.task
+        { method = "GET"
```

Comment:
Sorry, this must be `DELETE` as the name of this functions says. `delete` function also has this bug. This is my mistake.

Reply:
OMG, good call.